humble/distribution.yaml

```diff
@@ -11363,6 +11363,12 @@ repositories:
       url: https://github.com/ros-drivers/zbar_ros.git
       version: humble
     status: maintained
+  zed-ros2-interfaces:
+    source:
+      type: git
+      url: https://github.com/stereolabs/zed-ros2-interfaces.git
```

Comment:
I'm fine with this as-is, but I'll point out that this commit is actually the opposite of what we suggest nowadays: https://github.com/stereolabs/zed-ros2-interfaces/commit/7a914e4b787440f2e00d2ae1ed237645675ac019 .  That is, we recommend `foo_interfaces`, since that covers messages, services, and actions.

Regardless, I am going to go ahead and merge this since this is only the source entry; if you feel like it, you could revert that commit before you do your first real release.